aa_stripe/models.py

```diff
@@ -583,6 +594,35 @@ def _parse_coupon_notification(self, action):
         elif action == "deleted":
             StripeCoupon.objects.filter(coupon_id=coupon_id, created=created).delete()
 
+    def _parse_customer_source_notification(self, action):
+        stripe_card_id = self.raw_data["data"]["object"]["id"]
+        stripe_customer_id = self.raw_data["data"]["object"]["customer"]
+        try:
+            customer = StripeCustomer.objects.get(stripe_customer_id=stripe_customer_id)
+        except StripeCustomer.DoesNotExist:
+            raise StripeWebhookParseError(
+                _("There is no customer with id that is assigned to this card in our database"))
+        if action == "created":
+            try:
+                StripeCard.objects.all_with_deleted().get(stripe_card_id=stripe_card_id, customer=customer)
+            except StripeCard.DoesNotExist:
+                card = StripeCard(stripe_card_id=stripe_card_id, customer=customer)
+                if card.sync_with_stripe():
+                    card.save()
+                else:
+                    raise StripeWebhookParseError(_("Card with this id does not exists at Stripe API"))
+            else:
+                raise StripeWebhookParseError(_("Card with this id and customer already exists"))
```

Comment:
we'll have this error each time a new card gets created

Reply:
ok, i'll ignore this case